libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -1911,8 +1911,47 @@ string YulUtilFunctions::conversionFunction(Type const& _from, Type const& _to)
 			break;
 		}
 		case Type::Category::Struct:
-			solUnimplementedAssert(false, "Struct conversion not implemented.");
+		{
+			solAssert(toCategory == Type::Category::Struct, "");
+			auto const& fromStructType = dynamic_cast<StructType const &>(_from);
+			auto const& toStructType = dynamic_cast<StructType const &>(_to);
+
+			solUnimplementedAssert(!fromStructType.isDynamicallyEncoded(), "");
+			solUnimplementedAssert(toStructType.location() == DataLocation::Memory, "");
+			solUnimplementedAssert(fromStructType.location() == DataLocation::CallData, "");
+
+			MemberList::MemberMap toStructMembers = toStructType.nativeMembers(nullptr);
+			MemberList::MemberMap fromStructMembers = fromStructType.nativeMembers(nullptr);
+			solAssert(toStructMembers.size() == fromStructMembers.size(), "");
+
+			vector<map<string, string>> memberValuesAndSizes(toStructMembers.size());
+			vector<string> memberParamNames(toStructMembers.size());
+			for (size_t i = 0; i < toStructMembers.size(); ++i)
+			{
+				solAssert(toStructMembers[i].type->isImplicitlyConvertibleTo(*fromStructMembers[i].type), "");
```

Comment:
When implementing other conversion for structs, I realized that conversion between structs is only possible if structs has the same definition. So, this is not needed here. There is a need to add 
`solAssert(fromStructType.structDefinition() == toStructType.structDefinition(), "");`
at the beginning.